core/persistence-jpa-json/src/main/java/org/apache/syncope/core/persistence/jpa/dao/JPAJSONPlainSchemaDAO.java

```diff
@@ -31,6 +31,12 @@
         return Collections.emptyList();
     }
 
+    @Override
+    public <T extends PlainAttr<?>> boolean hasAttrs(PlainSchema schema, Class<T> plainAttrTable) {
+        // not possible
+        return true;
```

Comment:
this method should return `false` to be consistent with `hasAttrs()` returning an empty list.